packages/frontend-2/components/automate/runs/trigger-status/Dialog.vue

```diff
@@ -37,7 +37,7 @@
           size="sm"
           target="_blank"
           external
-          to="https://speckle.systems/blog/automate-with-speckle/"
```

Comment:
In theory the new blog will use the same url structure, so no need to update these